docs/releases/3.0.txt

```diff
@@ -329,6 +329,9 @@ Django 3.0, we're removing these APIs at this time.
 
 * ``django.utils.safestring.SafeBytes`` - Unused since Django 2.0.
 
+* ``django.utils.text.unescape_entities()`` - Use :func:`html.unescape`
```

Comment:
Django's `unescape_entities` had `@keep_lazy_text` applied. For Django's own use of the function in `multipartparser.py` I don't think it is needed, but maybe it's not a straight replacement for other users?